src/tox/helper/build_isolated.py

```diff
@@ -3,6 +3,9 @@
 dist_folder = sys.argv[1]
 backend_spec = sys.argv[2]
 backend_obj = sys.argv[3] if len(sys.argv) >= 4 else None
+backend_paths = sys.argv[4].split(':')
```

Comment:
@gaborbernat should I serialize this as JSON instead?

Reply:
I think it's fine, though you might want to check here if len is at least 4 👍

Reply:
@gaborbernat this script is always called with a fixed number of args in only one place in code. I'd say that the check above is also redundant.

My concern with `:` delimiter is that it's not escaped and paths having it would break things.